content/License.txt

```diff
@@ -0,0 +1,58 @@
+
```

Comment:
As per https://github.com/aspnet/libuv-package/pull/17#discussion_r107042701 - this file is not needed. The link in the nuspec has changed since.